Dockerfile

```diff
@@ -1,21 +1,17 @@
-FROM alpine:3.7
+FROM golang:1.11-alpine3.8
+RUN apk update
+RUN apk upgrade
 
-RUN apk update && apk add \ 
-	bash \ 
-	jq \ 
-	curl \ 
-	git \
-	tar \
-	gzip
+RUN apk --no-cache add jq bash git curl tar gzip
 
 # Add jfrog cli
 RUN curl -fL https://getcli.jfrog.io | sh \
     && mv ./jfrog /usr/local/bin/jfrog \
     && chmod 777 /usr/local/bin/jfrog
 
-RUN wget -qO- "https://packages.cloudfoundry.org/stable?release=linux64-binary&source=github" | tar -zx && \
-	mv cf /usr/local/bin && \
-	cf --version
+ADD https://cli.run.pivotal.io/stable?release=linux64-binary /tmp/cf-cli.tgz
```

Comment:
ADD from remote is discouraged best practice. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy 

> Because image size matters, using ADD to fetch packages from remote URLs is strongly discouraged; you should use curl or wget instead. That way you can delete the files you no longer need after they’ve been extracted and you don’t have to add another layer in your image. For example, you should avoid doing things like: